### PR TITLE
THE JUKENING OF OUR TIME

### PR DESCRIPTION
**Pull request by Kharkov72**
The juke playlists have been updated, however those need to be on pomfs end.
This PR is for some codeside stuff and will be merged at the same time.
If you need to do something you gotta do it yourself.
Oh GOD Oh FUCK.
[help] [sound]

Untitled

:cl:
 * rscadd: THE JUKE REWORK IS HERE AAAAA

### CODE REVIEW

**Comment by SNilWillIllShill** [comment]
This is it. This is the culmination of 4 months of almost nonstop working on my end, and about one or two months on the end of my helpers. We desperately need coder help on the last few bits, but otherwise we are basically ready to merge.

I hope you enjoy the fruits of our labor. I know I certainly will.

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 1]
```diff
@@ -299,6 +299,7 @@ var/veil_thickness = CULT_PROLOGUE
 				..()
 				command_alert(/datum/command_alert/bloodstones_raised)
 				new_obj = new /datum/objective/bloodcult_bloodbath
+				ticker.StartThematic("endgame")
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm]: This is literally the exact opposite of what I told you to do. Implement this in an OOP way.

Comment (Kharkov72) [reply code/datums/gamemode/factions/bloodcult/bloodcult.dm]: I think I got it chief

File: code/datums/gamemode/factions/faction.dm [hunk 2]
```diff
@@ -236,7 +237,10 @@ var/list/factions_with_hud_icons = list()
 			set_security_level("blue")
 			ticker.StopThematic()
 		if(FACTION_ENDGAME) //Faction is nearing victory. Set red alert and play endgame music.
-			ticker.StartThematic("endgame")
+			if(playlist)
+ 			ticker.StartThematic(playlist)
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/faction.dm]: needs tabbing

File: code/datums/gamemode/factions/faction.dm [hunk 3]
```diff
@@ -236,7 +237,10 @@ var/list/factions_with_hud_icons = list()
...
+			else
+ 			ticker.StartThematic("endgame")
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/faction.dm]: needs tabbing

**Comment by CptWad** [comment]
obligatory dance thread

**Comment by Shadowmech88** [comment]
It looks like the vidya playlists have been removed entirely.
Has some/all of that music been redistributed into other playlists based on what kind of music it is beyond "is from a video game?"
Or has all the music from video games simply been removed?

**Comment by yclatious** [comment]
As one of the helpers, Id merely like to say that Nexis is a cuckhold for making the juke as shit as it was and that the effort to uncuck it from the fat hands of that furry shit took much, much longer than I ever thought it could take. But it was worth it to make the juke much, much better, and hopefully you lads will agree.

Also I think some of the songs got moved but other than that, it got the fat cull. Its to be expected, knowing the origin of the playlists and the fact each vidya was triple digits big.

**Comment by MadmanMartian** [comment]
Do the vinyls actually work now?

**Comment by Kharkov72** [comment]
Should work, but mapped jukeboxes are weird sometimes.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
But is battle of lil' slugger still default?

**Comment by KingofHONK** [comment]
if battle of little slugger is not on every playlist I am thumbsing down this PR and ban requesting each of you

**Comment by Kharkov72** [comment]
_It's in emagged now_
Actually it's in rock and not first to my knowledge. 🤷‍♂

**Comment by Irradiation** [comment]
DELET

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 4]
```diff
@@ -787,7 +787,7 @@ obj/item/asteroid/basilisk_hide/New()
 		for(var/i = 0 to rand(24,46))
 			new possessed_ore(src.loc)
 
-	new /obj/item/weapon/vinyl/rock(src.loc) //It is a rock monster after all
```
Comment (MadmanMartian) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:790]: 👍

Comment (Kharkov72) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:790]: I like the pun but every juke already has rock and to make this vinyl something of value you'd need to remove rock from the jukeboxes which is dumb.
Curse whoever did this originally.

**Comment by FudgePucker** [comment]
Hell yeah

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm (hunk 4 above)
Comment (MadmanMartian) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:790]: I was the one who did this originally. I was mistaken and thought rock was only on the DJ satellite.

**Comment by Theophrastus1** [comment]
the PR that killed /vg/

**Comment by Pastamadness** [comment]
So true.

**Pull request merged**